src/theme/sass-partials/_input.scss

```diff
@@ -36,3 +37,10 @@ ion-select.select {
     color: inherit;
   }
 }
+
+ion-input.input-ios .text-input-clear-icon {
+  right: 16px;
+  background-image: url("data:image/svg+xml;charset=utf-8,<svg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20512%20512'><polygon%20fill='%23222222'%20points='405,136.798%20375.202,107%20256,226.202%20136.798,107%20107,136.798%20226.202,256%20107,375.202%20136.798,405%20256,285.798%20375.202,405%20405,375.202%20285.798,256'/></svg>");
```

Comment:
Does this work offline? - think the image will need to be stored in the app

Reply:
The image is an SVG, so it's just data :)

Reply:
Paste it into your (offline) browser window to see `data:image/svg+xml;charset=utf-8,<svg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20512%20512'><polygon%20fill='%23222222'%20points='405,136.798%20375.202,107%20256,226.202%20136.798,107%20107,136.798%20226.202,256%20107,375.202%20136.798,405%20256,285.798%20375.202,405%20405,375.202%20285.798,256'/></svg>`

Reply:
Would prefer to have it stored in `src/assets/imgs` alongside all the other images we use in the app which would also make it reusable. Then this reference would just be `assets/clear-icon.svg` or something along those lines.

Reply:
@buzzub - Didn't know that, learn new things

The cross isn't a proper cross though , theres something wrong with it when you open it up big

Reply:
> Would prefer to have it stored in `src/assets/imgs` 
Yeah, makes sense. In this instance I copied the CSS from the MD ionic styling, which had the image inline.